README.md

```diff
@@ -1,5 +1,54 @@
 # curriculum-cr-training-contact-form
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+# Contact form
+
+> This project consists on Builting a responsive Portfolio from the Mobile version to the Desktop version, using Html and Css.
+
+
+## Built With
+
+- Vs code
+- Html
+- Css
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] Please remember to remove the Live Demo section here :point_up: , since the link in it doesn't work :sparkles: It's essential to keep our README very professional :smile: